src/emmodel/model.py

```diff
@@ -69,10 +69,10 @@ def get_coefs_df(self) -> pd.DataFrame:
                         variable_names.extend([f"{variable.name}_{i}"
                                                for i in range(variable.size)])
             coefs.update({
-                variable_name: self.results[i]["coefs"][j]
+                variable_name: [self.results[i]["coefs"][j], np.sqrt(self.results[i]["vcov"][j,j])]
                 for j, variable_name in enumerate(variable_names)
             })
-        return pd.DataFrame(coefs, index=[0])
+        return pd.DataFrame(coefs, index=['Estimate','SE'])
```

Comment:
Thanks @jiaweih this looks great! I am thinking maybe we should reshape this long instead of wide, which means that
```
| coef_name | coef_value | coef_sd |
| intercept | 1.0 | 0.1 |
```
And we could add another column called `model_id` to encode which model/stage it comes from.

Reply:
I have reshaped the dataframe and add a new column, but not quite sure about the model_id.  https://github.com/ihmeuw-msca/emmodel/pull/43/commits/e927bdd50b52490a71c837a42d7a4682599a4e3a